images/macos/provision/core/nvm.sh

```diff
@@ -11,18 +11,9 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/$VERSION/install.sh | bash
 if [ $? -eq 0 ]; then
         . ~/.bashrc
         nvm --version
-        nvm install lts/boron # 6.x
-        nvm install lts/carbon # 8.x
         nvm install lts/dubnium # 10.x
         nvm install lts/erbium # 12.x
-        nvm install v13 # 13.x
         nvm install v14 # 14.x
```

Comment:
Does it make sense to use a version number instead of codename for Node 10 and 14 and put all Node versions to an array to avoid adding a new line for each version? Something like this:
```
nodeVersions=("v10" "v12" "v14")
for version in ${nodeVersions[@]}; do
  nvm install $version
done
```

Reply:
Thank you. Done.